bonus.md

```diff
@@ -3,11 +3,16 @@
 Edit this file to answer the following questions. Give examples. You may need to do a bit of research. :)
 
 1. Why is Hash.fetch useful?
+-Hash.fetch still works if the default value you supply is a boolean, while hash[:key] does not because it returns nil which is falsey.
 
 2. What is the disadvantage of single inheritance, and what's a possible solution to that problem?
+-Compared to composition, inheritance sometimes overcomplicates things and is not conducive to single responsibility classes. One solution to this is to create a module and include it in all relevant classes.
```

Comment:
So, I think this question is actually about single vs multiple inheritance, which ruby does not support.  The answer is that where you'd use multiple inheritance in other languages, in Ruby you'll use Modules.

In general, I'd say that Inheritance is *more* conducive to single responsibility than including modules.